File-Formats/DocIO/Create-Word-document-in-Console-application.md

```diff
@@ -0,0 +1,533 @@
+    //Append paragraph.
+    paragraph = table[2, 1].AddParagraph();
+    paragraph.ParagraphFormat.AfterSpacing = 0;
+    paragraph.ParagraphFormat.LineSpacing = 12f;
+    paragraph.BreakCharacterFormat.FontSize = 12f;
+    paragraph.BreakCharacterFormat.FontName = "Times New Roman";
+    textRange = paragraph.AppendText("Product No: BK-R93R-44\r") as WTextRange;
+    textRange.CharacterFormat.FontSize = 12f;
+    textRange.CharacterFormat.FontName = "Times New Roman";
+    textRange = paragraph.AppendText("Size: 44\r") as WTextRange;
+    textRange.CharacterFormat.FontSize = 12f;
+    textRange.CharacterFormat.FontName = "Times New Roman";
+    textRange = paragraph.AppendText("Weight: 14\r") as WTextRange;
+    textRange.CharacterFormat.FontSize = 12f;
+    textRange.CharacterFormat.FontName = "Times New Roman";
+    textRange = paragraph.AppendText("Price: $3,578.27\r") as WTextRange;
+    textRange.CharacterFormat.FontSize = 12f;
+    textRange.CharacterFormat.FontName = "Times New Roman";
+    //Append paragraph.
+    section.AddParagraph();
+
+    //Create FileStream to save the Word file.
+    using (FileStream outputStream = new FileStream("Result.docx", FileMode.Create, FileAccess.ReadWrite, FileShare.ReadWrite))
+    {
+        //Save the Word file.
+        document.Save(outputStream, FormatType.Docx);
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+You can download a complete working sample from GitHub.
+
+By executing the program, you will get the **Word document** as follows.
+
+![.NET Core output Word document](Mac-images/GettingStartedOutput.jpg)
+
+Click [here](https://www.syncfusion.com/document-processing/word-framework/net-core) to explore the rich set of Syncfusion Word library (DocIO) features. 
+
+An online sample link to [create a Word document](https://ej2.syncfusion.com/aspnetcore/Word/HelloWorld#/material3) in ASP.NET Core. 
+
+## Create Word document in .NET Framework
+
+The below steps illustrates creating a simple **Word document** in console application using **.NET Framework**.
+
+Step 1: Create a new **Console application** project.
+![Create a C# Console application in Visual Studio](Console-Images/.NET-FrameWork/Console-Template-Net-FrameWork.png)
+
+Step 2: Install [Syncfusion.DocIO.WinForms](https://www.nuget.org/packages/Syncfusion.DocIO.WinForms/) NuGet package as a reference to your Windows Forms application from the [NuGet.org](https://www.nuget.org/).
+
+![Install Syncfusion.DocIO.WinForms NuGet package](Console-Images/.NET-FrameWork/Nuget-Package-NET-FrameWork.png)
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering Syncfusion license key in your application to use our components.
```

Comment:
Add the Note about WinForms NuGet as 1st point.
2nd point - you can add license related note

Reply:
Added